src/Bigtable/composer.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "google/cloud-bigtable",
+    "description": "Cloud Bigtable Client for PHP",
+    "license": "Apache-2.0",
+    "minimum-stability": "stable",
+    "require": {
+        "google/cloud-core": "^1.0",
```

Comment:
We shouldn't need to require core, rather, `"ext-grpc": "*"`

Reply:
Done.